app/controllers/profile_orders_controller.rb

```diff
@@ -0,0 +1,9 @@
+class ProfileOrdersController < ApplicationController
+  def show
+    @order = current_user.orders.last
```

Comment:
I think this may get converted into an index page in a future story--at which point creating that will be unnecessary--so I'll hold off on doing that for now, but I agree with you if we do stick with a show page.